### DISCUSSION TITLE
Hovering over the values could show full path rather than just variable name

### DISCUSSION THREAD

**Discussion by byte-arcane** [WerWolv/ImHex#1726] [Feedback]
As per title. Sometimes I have a vector3 struct in a class in another class, and it would be useful to see e.g. foo.bar.y when hovering, instead of just y (btw fantastic software, thanks!)

![image](https://github.com/WerWolv/ImHex/assets/83374486/5ddb1503-f696-4085-b06a-b7b9a32749ea)

**Comment by WerWolv** [comment]
Latest commit adds the full path to the tooltip when you hold down shift :)
![image](https://github.com/WerWolv/ImHex/assets/10835354/122bf56f-c7e2-4ea5-ad66-92176e9378da)

**Comment by byte-arcane** [comment]
That's amazing, thanks!
